lib/rucio/common/schema/atlas.py

```diff
@@ -58,7 +60,8 @@
 
 NAME = {"description": "Data Identifier name",
         "type": "string",
-        "pattern": "^[A-Za-z0-9][A-Za-z0-9\\.\\-\\_]{1,%s}$" % NAME_LENGTH}
+        "maxLength": NAME_LENGTH,
+        "pattern": "^[A-Za-z0-9][A-Za-z0-9\\.\\-\\_]*$"}
```

Comment:
just to clarify - is there a reason why there's `*` at the end of the pattern here, compared to having `+` like for `SCOPE` and `ACCOUNT`?

for reference:
- `* matches the previous token between zero and unlimited times, as many times as possible, giving back as needed (greedy)`
- `+ matches the previous token between one and unlimited times, as many times as possible, giving back as needed (greedy)`

Reply:
I defined all regexes in a way that they require at least 1 character to be present.

So for regexes that define a single character class, I used `+`.

For regexes that use an smaller, initial character set for the first letter and then a wider range for any following characters, I used `*`